src/stan/model/indexing/rvalue.hpp

```diff
@@ -717,9 +722,9 @@ inline auto rvalue(
     const cons_index_list<Idx, cons_index_list<index_min_max, nil_index_list>>&
         idxs,
     const char* name = "ANON", int depth = 0) {
-  math::check_range("matrix[..., min_max] indexing", name, x.rows(),
+  math::check_range("matrix[..., min_max] min column indexing", name, x.cols(),
                     idxs.tail_.head_.min_);
-  math::check_range("matrix[..., min_max] indexing", name, x.rows(),
+  math::check_range("matrix[..., min_max] max column indexing", name, x.cols(),
```

Comment:
More bugz?

Reply:
Yes this was checking rows when it should have checked columns. It was passing tests but in the wrong place